bookkeeper-server/build.gradle

```diff
@@ -18,6 +18,8 @@
  */
 plugins {
     id 'java'
+    id 'com.adarshr.test-logger'
+    //id 'org.gradle.test-retry'
```

Comment:
we can remove if not used for now

Reply:
yeah

Reply:
@pkumar-singh - could you update the PR to remove at least this line?

Reply:
Sure done.